### PR TITLE
Change tests for Android and add eme

### PR DESCRIPTION
**Pull request by marcosholgado**
https://app.asana.com/0/1125189844152671/1201514178819251

This PR adds eme as a new test only for Android and it excludes Android from the autofill tests (not yet implemented) and trackingCookies1p

### CODE REVIEW

File: privacy-configuration/tests.json [hunk 1]
```diff
@@ -175,6 +176,19 @@
...
+            },
+            {
+                "name": "eme should be disabled",
+                "featureName": "eme",
+                "siteURL": "https://example.org",
+                "expectFeatureEnabled": false,
+                "exceptPlatforms": [
+                    "safari-extension",
+                    "ios-browser",
+                    "macos-browser",
```
Comment (kdzwinel) [inline privacy-configuration/tests.json:191]: Web extension also doesn't have that feature AFAIK:

```suggestion
                    "macos-browser",
                    "web-extension",
```

File: privacy-configuration/tests.json [hunk 2]
```diff
@@ -356,6 +371,19 @@
...
+            },
+            {
+                "name": "eme should be enabled",
+                "featureName": "eme",
+                "siteURL": "https://example.org",
+                "expectFeatureEnabled": true,
+                "exceptPlatforms": [
+                    "safari-extension",
+                    "ios-browser",
+                    "macos-browser",
```
Comment (kdzwinel) [inline privacy-configuration/tests.json:386]: ditto

```suggestion
                    "macos-browser",
                    "web-extension",
```

File: privacy-configuration/tests.json [hunk 3]
```diff
@@ -356,6 +371,19 @@
                 "expectFeatureEnabled": true,
                 "exceptPlatforms": [
                     "safari-extension",
+                    "windows-browser",
+                    "android-browser"
```
Comment (kdzwinel) [inline privacy-configuration/tests.json:375]: do you have a task for implementing autofill on android?

**Review by kdzwinel — COMMENTED** [review]
Thanks, this looks good! Found some nits

File: privacy-configuration/tests.json (hunk 3 above)
Comment (marcosholgado) [reply privacy-configuration/tests.json:375]: Aye -> https://app.asana.com/0/414730916066338/1201514178819254

File: privacy-configuration/tests.json (hunk 2 above)
Comment (marcosholgado) [reply privacy-configuration/tests.json:387]: Done!

File: privacy-configuration/tests.json (hunk 1 above)
Comment (marcosholgado) [reply privacy-configuration/tests.json:191]: Done!

**Review by kdzwinel — APPROVED** [review]
PERFECTION

**Pull request merged**